src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/index.scss"
```

Comment:
The href attribute is pointing to a SCSS file. Typically, browsers cannot directly interpret SCSS files. You should link to a compiled CSS file instead.